### ISSUE TITLE
[Bug]: New devices aren't showing in control panel UI, they exist in conf and HA

### ISSUE THREAD

**Issue by ldymek** [zwave-js/zwavejs2mqtt#2250]
### Checklist

- [X]  I am not using Home Assistant. Or: a developer has told me to come here.
- [X] I have checked the troubleshooting section and my problem is not described there.
- [X] I have read the changelog and my problem is not mentioned there.

### Deploy method

Docker

### Zwavejs2Mqtt version

6.2.0

### ZwaveJS version

8.9.1

### Describe the bug

When I am adding new device it is added successfully but not showing in control panel UI. It is visible under nodes dump and directly in Home Assistant

### To Reproduce

Add new device

### Expected behavior

Device should appear in UI

### Additional context

Sometimes Heal network option is restoring back missing devices to the UI. In the UI nodes 1 to 24 are visible, the rest is hidden. **Restart app or even whole OS, update zwavejs2mqttt is reverting back visible nodes in the UI.** 

Nodes are visible in netowrk graph.

[nodes_dump_08_02_2022.zip](https://github.com/zwave-js/zwavejs2mqtt/files/8027763/nodes_dump_08_02_2022.zip)

**Comment by robertsLando** [comment]
@ldymek Maybe you simply have to increese nodes per page shown in table or go to next page? Check table footer, it provides paging functionalities

**Comment by robertsLando** [comment]
@ldymek Could you try using latest 6.5.1?

**Comment by ldymek** [comment]
I updated it and at the beggining I saw only 24 devices but now I see them all, I've even added new device and it remains on UI. So here's the bottom of the list:

![image](https://user-images.githubusercontent.com/56914130/153360223-344f7e1f-2af7-4ec0-ac4f-eee4dffcc3b6.png)

I decided to reboot the host (raspberry pi 3b+) and after reboot I still have them all, after that I've pulled power cord from it and booted again. The issue seems to be resolved somehow. Did 6.5.1 version changed something or this is just coincidence?
